docs/how-to-guides/block-tutorial/applying-styles-with-stylesheets.md

```diff
@@ -58,8 +58,8 @@ registerBlockType( 'gutenberg-examples/example-02-stylesheets', {
 {% Plain %}
 
 ```js
-( function ( blocks, element, blockEditor ) {
-	var el = element.createElement;
+( function ( blocks, React, blockEditor ) {
```

Comment:
Is uppercase also how the default React scripts adds the window variable?

Reply:
yes